index.html

```diff
@@ -12,28 +12,215 @@
+        <p id="details">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
             don’t hestiate to contact me.</p>
+            </div>
             <div class="social-media">
                 <ul class="social-media-icons">
-                    <li id="github"><a href="#"><img src="images/Normal Button/Tertiary/Icons/Icon -GitHub.png" alt="github"></a></li>
-                    <li id="lkdn"><a href="#"><img src="images/Normal Button/Tertiary/Icons/Normal Button/Tertiary/Icons/Normal Button/Tertiary/Icons/Icon - Linkedin.png" alt="linkedin"></a></li>
-                    <li id="ang"><a href="#"><img src="images/Normal Button/Tertiary/Icons/Icon - Angelist.png" alt="angellist"></a></li>
-                    <li id="twt"><a href="#"><img src="images/Normal Button/Tertiary/Icons/Icon - Twitter.png" alt="twitter"></a></li>
-                    <li id="mdm"><a href="#"><img src="images/Normal Button/Tertiary/Icons/Icon - Medium.png" alt="medium"></a></li>
+                    <li id="github"><a href="https://github.com/waseemqureshi2020"><img src="images/Normal Button/Tertiary/Icons/Icon -GitHub.png" alt="github"></a></li>
+                    <li id="lkdn"><a href="https://www.linkedin.com/in/waseem-qureshi-2231741a2/"><img src="images/Normal Button/Tertiary/Icons/Normal Button/Tertiary/Icons/Normal Button/Tertiary/Icons/Icon - Linkedin.png" alt="linkedin"></a></li>
+                    <li id="ang"><a href="https://angel.co/u/waseem-qureshi-1"><img src="images/Normal Button/Tertiary/Icons/Icon - Angelist.png" alt="angellist"></a></li>
+                    <li id="twt"><a href="https://twitter.com/WASEEMQ2020"><img src="images/Normal Button/Tertiary/Icons/Icon - Twitter.png" alt="twitter"></a></li>
+                    <li id="mdm"><a href="https://medium.com/@waseemqureshiofficial2020"><img src="images/Normal Button/Tertiary/Icons/Icon - Medium.png" alt="medium"></a></li>
                 </ul>
             </div>
+            </div>
+            <div class="new-bgi">
+        </div>
+            </div>
     </section>
-    <section>
-       <img class="illu" src="images/illustration-header-mobile-1.png"></div>
-    </section>
+    <section class="work-section">
+        <div class="work-container">
+            <div><h1 class="recent-work">My Recent Works</h1></div>
+       <div class="work-image">
+           <img src="images/My Work Images/Img Placeholder.png" alt="work">
+           </div>
+           <div class="work-content">
+            <h2 class="post-type1">Multi-Post Stories</h2>
+            <p class="para-type1">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text. </p>
```

Comment:
- [ ] Kindly give your `multi-post stories` section a margin to match the Figma design `hint`.  use `margin-left`  and `margin-right`.

![image](https://user-images.githubusercontent.com/98453979/192098431-95b679b8-c145-409c-950e-53dbd43e0ff3.png)
 
### Figma Design

![image](https://user-images.githubusercontent.com/98453979/192098479-3d2275be-b338-4926-bb0b-e2890eb6f9be.png)

- [ ] Kindly style your buttons border to match the figma design.  You can use `border: 1px solid color`

![image](https://user-images.githubusercontent.com/98453979/192098578-06329583-b704-406d-84a5-7681cce4aad9.png)

![image](https://user-images.githubusercontent.com/98453979/192098661-69e3028e-9fff-4696-90db-228377c68f41.png)

Reply:
I have changed CSS properties for both of these sections, Kindly have a look at the design now.

Reply:
Good work @waseemqureshi2020 for making the changes👍 . I see it has already been approved